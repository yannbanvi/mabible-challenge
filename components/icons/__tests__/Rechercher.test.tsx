/** @jest-environment jsdom */
import '@testing-library/jest-dom'
import { render } from '@testing-library/react'
import RechercheIcon from '../Recherche'
describe('RechercheIcon', () => {
  it('renders RechercheIcon page successfully', () => {
    const { container } = render(<RechercheIcon />)
    const svg = container.getElementsByTagName('svg');
    expect(svg).toHaveLength(1);
    expect(container).toMatchSnapshot();
  })
});