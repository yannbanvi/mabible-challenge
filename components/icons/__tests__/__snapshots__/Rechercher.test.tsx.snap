// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RechercheIcon renders RechercheIcon page successfully 1`] = `
<div>
  <svg
    fill="none"
    height="12"
    viewBox="0 0 12 12"
    width="12"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      clip-path="url(#clip0_524_2182)"
    >
      <path
        clip-rule="evenodd"
        d="M5.26123 1.09991C2.96299 1.09991 1.09991 2.96299 1.09991 5.26123C1.09991 7.55946 2.96299 9.42255 5.26123 9.42255C7.55946 9.42255 9.42255 7.55946 9.42255 5.26123C9.42255 2.963 7.55946 1.09991 5.26123 1.09991ZM0 5.26123C0 2.35553 2.35553 0 5.26123 0C8.16692 0 10.5225 2.35553 10.5225 5.26123C10.5225 6.51598 10.0832 7.66815 9.3501 8.57235L11.8389 11.0612C12.0537 11.2759 12.0537 11.6242 11.8389 11.8389C11.6242 12.0537 11.2759 12.0537 11.0612 11.8389L8.57234 9.3501C7.66815 10.0832 6.51598 10.5225 5.26123 10.5225C2.35553 10.5225 0 8.16692 0 5.26123Z"
        fill="#211B28"
        fill-rule="evenodd"
      />
    </g>
    <defs>
      <clippath
        id="clip0_524_2182"
      >
        <rect
          fill="white"
          height="12"
          width="12"
        />
      </clippath>
    </defs>
  </svg>
</div>
`;
